### PR TITLE
[RPC] correct non PIVXified help commands texts and add missing "ischange" result.

### PR DESCRIPTION
**Pull request by furszy**
Straightforward:
Corrected few remaining non-PIVXified help commands texts and added missing "ischange" result return (for transparent addresses) in `getaddressinfo`.

### CODE REVIEW

File: src/wallet/rpcwallet.cpp [hunk 1]
```diff
@@ -200,7 +200,9 @@ UniValue getaddressinfo(const JSONRPCRequest& request)
     ret.pushKV("isshield", pTransDest == nullptr);
 
     if (pTransDest) {
-        ret.pushKV("scriptPubKey", HexStr(GetScriptForDestination(*pTransDest)));
+        CScript scriptPubKey = GetScriptForDestination(*pTransDest);
+        ret.pushKV("scriptPubKey", HexStr(scriptPubKey));
+        ret.pushKV("ischange", pwallet->IsChange(scriptPubKey));
```
Comment (random-zebra) [inline src/wallet/rpcwallet.cpp:205]: "ischange" should be added in the help too.

Comment (furszy) [reply src/wallet/rpcwallet.cpp:206]: true, updated.

**Review by random-zebra — APPROVED** [review]
utACK ba6fc50a0e9a91b477dd4a6178db6e82156bcc1f

**Review by Fuzzbawls — APPROVED** [review]
utACK ba6fc50a0e9a91b477dd4a6178db6e82156bcc1f

**Pull request merged**